flocx_market/tests/unit/api/test_app.py

```diff
@@ -33,3 +33,17 @@ def test_root_data(client):
         }
     }
     assert json.loads(response.data) == version
+
+
+@mock.patch('flocx_market.cmd.api.flocx_market_service.prepare_service')
+@mock.patch('flocx_market.cmd.api.service.ProcessLauncher.launch_service')
+@mock.patch('flocx_market.cmd.api.service.ProcessLauncher.wait')
+def test_keystone_app(wait, launch, prepare):
+    CONF.clear()
+    CONF.set_override("auth_enable", True,
+                      group='api')
+
+    main.main()
+
+    launch.assert_called_once()
+    wait.assert_called_once()
```

Comment:
This is definitely a good test to have. I think it should be moved to tests/unit/cmd/test_api.py, however.